flatpak-proxy.c

```diff
@@ -1924,11 +1924,27 @@ is_dbus_method_call (Header *header)
 }
 
 static gboolean
-is_introspection_call (Header *header)
+is_properties_call (Header *header)
 {
   return
     header->type == G_DBUS_MESSAGE_TYPE_METHOD_CALL &&
-    g_strcmp0 (header->interface, "org.freedesktop.DBus.Introspectable") == 0;
+    g_strcmp0 (header->interface, "org.freedesktop.DBus.Properties") == 0;
```

Comment:
I'm not very familiar with this project, so just a question, but shouldn't this require permissions as it can `Set` properties?